databases/core.py

```diff
@@ -351,7 +351,10 @@ def netloc(self) -> typing.Optional[str]:
 
     @property
     def database(self) -> str:
-        return self.components.path.lstrip("/")
+        path = self.components.path
+        if path[:1] == "/":
+            return path[1:]
+        return path
```

Comment:
This is not covered.  Let me know if you want to take this in the first place.

OTOH I am not using SQLite anymore, so feel free to just close it also.. ;)

Reply:
I reckon either you or I oughta follow up on it given it's now also being reported elsewhere, yup. #115

Happy to pick it up if you've moved on for it.